wpilib/wpilib/interruptablesensorbase.py

```diff
@@ -1,26 +1,33 @@
-# validated: 2017-11-19 EN b65447b6f5a8 edu/wpi/first/wpilibj/InterruptableSensorBase.java
-#----------------------------------------------------------------------------
+# validated: 2017-11-23 TW b65447b6f5a8 edu/wpi/first/wpilibj/InterruptableSensorBase.java
+# ----------------------------------------------------------------------------
 # Copyright (c) FIRST 2008-2012. All Rights Reserved.
 # Open Source Software - may be modified and shared by FRC teams. The code
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
-#----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
 
 import hal
 import weakref
+import enum
 
 from .sensorbase import SensorBase
 
 __all__ = ["InterruptableSensorBase"]
 
+
 class InterruptableSensorBase(SensorBase):
     """Base for sensors to be used with interrupts"""
 
+    class WaitResult(enum.IntEnum):
+        kTimeout = 0x0
```

Comment:
Style thought, I would probably pad each of these out to 4 digits... `0x0000`, `0x0100`, etc.